.ci.yaml

```diff
@@ -52,6 +52,8 @@ targets:
     recipe: engine_v2/engine_v2
     properties:
       config_name: local_engine
+    # local_engine schedules a bunch of other builds, so it's likely to timeout waiting for those builds to run.
+    timeout: 90
```

Comment:
go even longer, 180?

The shards it is scheduling have their own timeouts too.

Reply:
Done.